esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2717,7 +2717,7 @@ bool CFileSprayEx::onFileList(IEspContext &context, IEspFileListRequest &req, IE
         if (!isEmptyString(fileNameMask))
         {
             const char* ext = pathExtension(sPath.str());
-            if (!strieq(ext, "cfg") && !strieq(ext, "log"))
+            if (ext && !strieq(ext, "cfg") && !strieq(ext, "log"))
```

Comment:
The code here is to check the user's input inside the 'Path'. If the input contains a file extension (ex. Path=.../*.log) and the extension is not cfg and not log, the code throws the exception.